samples/http_2.js

```diff
@@ -0,0 +1,10 @@
+import http from "k6/http";
+import { check } from "k6";
+
+export default function() {
+  const HOMEPAGE_RESPONSE = http.get("https://www.bbc.co.uk/");
```

Comment:
Now you're just making the request twice.

Reply:
Fails with 'h2'.

Reply:
Sorry. I will amend on squash.